python/runtime/cudaq/operators/py_matrix_op.h

```diff
@@ -0,0 +1,17 @@
+/****************************************************************-*- C++ -*-****
+ * Copyright (c) 2022 - 2025 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
```

Comment:
Some new header files don't have a `#pragma once` guard. Although we don't expect them to be included in any other header files, it might be good to add the include guard for consistency.

Reply:
py_spin_op.h also didn't have it. I tend to only add pragma once when including the header multiple times would cause issues (e.g. the header contains more than declarations only), which is not the case here. Is it best practice to always have pragma once? I thought it might be more hassle for the compiler and cause more issues than it solves if it is fine to have it included multiple times.